src/opm/output/eclipse/Summary.cpp

```diff
@@ -576,6 +577,10 @@ template< rt phase, bool injection = true >
 inline quantity rate( const fn_args& args ) {
     double sum = 0.0;
 
+    std::string tracer_name = "";
+    if ( phase == rt::tracer && args.keyword_name.size() > 4) 
+        tracer_name = args.keyword_name.substr(4,std::string::npos);
```

Comment:
You don't need to pass `std::string::npos` here.  That is the default value for the second parameter.